src/DebuggerForm.cpp

```diff
@@ -1535,3 +1505,33 @@ void DebuggerForm::addressSlot(int addr, qint8& ps, qint8& ss, qint16& segment)
 			segment = memLayout.romBlock[q];
 	}
 }
+
+void DebuggerForm::reloadBreakpoints(bool merge)
+{
+	auto command = new Command("debug_list_all_breaks",
+		[this, merge](const QString& message){ emit replyReceived(message, merge); }
```

Comment:
This happened because `replyReceived` was supposed to do more stuff. But then I've broken it down further by creating `breakStateEndered` and `runStateEntered`.